src/tests/signup.test.js

```diff
@@ -22,7 +24,7 @@ describe('Barefoot nomad signup tests', () => {
       });
   });
 
-  it('users should be able to verify their accounts', (done) => {
+  it.skip('users should be able to verify their accounts', (done) => {
```

Comment:
```suggestion
  it('users should be able to verify their accounts', (done) => {
```
Why did you skip this? better use dummy data to insert user with a verified account